designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentService.scala

```diff
@@ -274,6 +283,21 @@ class DeploymentService(
     actionsDefinedInState ++ actionsDefinedInCustomActions
   }
 
+  // TODO: rename CustomActionDefinition to ScenarioActionDefinition
```

Comment:
We could leave `CustomActionDefinition` as it is and create a new `ScenarioActionDefinition` which has implementation for both custom actions and fixed actions. Thanks to that we won't need to create fake empty `allowedStateStatusNames`, fake icon etc. WDYT?